packages/shared-ux/chrome/navigation/src/ui/components/navigation_item_open_panel.tsx

```diff
@@ -81,13 +81,18 @@ export const NavigationItemOpenPanel: FC<Props> = ({ item, navigateToUrl, active
   const onLinkClick = useCallback(
     (e: React.MouseEvent) => {
       if (!href) {
+        if (selectedNode?.id === item.id) {
```

Comment:
When I click on the link (`Applications` in my video example) toggling doesn't work. The panel opens and then doesn't close. I'll further look into it, but if you have an idea what's going wrong here, let me know.

https://github.com/user-attachments/assets/1a4c36ae-dca2-46fe-ac52-fbc48fd9efb2

Reply:
This [commit](https://github.com/elastic/kibana/pull/192805/commits/412622884c7060ef677311fc17fe0f71caefc905) fixes the toggle issue. Thanks @sebelga for helping out!